src/app/newSettings/Sections/Account/Plans/utils/planUtils.ts

```diff
@@ -46,26 +48,30 @@ const getCurrentChangePlanType = ({
   return 'free';
 };
 
-const getPlanName = (storagePlan: StoragePlan | null) => {
-  return storagePlan?.simpleName ?? FreeStoragePlan.simpleName;
+const getPlanName = (storagePlan: StoragePlan | null, limit?: number) => {
+  if (storagePlan?.simpleName) return storagePlan?.simpleName;
+  if (limit) return bytesToString(limit, false);
```

Comment:
why return the parsed limit as the plan name? are there any cases where a plan does not have a plan name?

Reply:
Yes, in my case I got an empty value because the subscription does not have the `simpleName` field.